Initializes the ethical marketing funnel post (#188)
readthedocs/ethicalads.io: content/posts/2021-marketing-funnel.md

Title: Building an Ethical Marketing Funnel
Date: September 30, 2021
description:
tags: marketing, privacy
authors: Ra Cohen
image: /images/posts/2021-marketing-funnel.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@campaign_creators?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Campaign Creators</a> on <a href="https://unsplash.com/s/photos/funnel?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

> ## The Problem

Reviewer: Were we using sentence case on these H2s? Or title case?

When growing an online business, it's important to consider 
where your prospects are coming from and how to convert these prospects into customers. 
One key insight into this conversion comes from observing how your users interact with your site. 
Without some understanding of why people click on your page, what they do while they're there, 
and how this behavior corresponds to paying customers, 
you're at a serious disadvantage when it comes to optimizing your site to grow your business. 
Part of this process is building marketing funnel. 

A marketing funnel is a continuous mission of getting, keeping, and growing your customer base 
while using technology to meet their needs more efficiently. 
However, many implementations of marketing funnels execute this mission 
by collecting and hoarding personally identifiable information of individual customers. 
This is by no means required by the definition nor is it even necessary 
as marketing funnels have existed long before web browser trackers let alone the internet. 

> EthicalAds is a small start-up currently focusing on expanding our userbase which comes in two segments. 

Reviewer: ```suggestion

## Our Business

EthicalAds is a small start-up currently focusing on expanding our userbase which comes in two segments. 
```

like search engines or the "Ad served Ethically" text which runs beneath every ad. 
However, we previously had no good way of determining the relative success of these various approaches. 
In order to prioritize our time on new ads, landing pages, or other marketing strategies, 
we wanted data to guide our decision making to help us meet the needs of our customers efficiently. 
In other words, we needed a marketing funnel. 
Specifically, we wanted our marketing funnel to be privacy-preserving, open-source, and transparent. 
In a word, Ethical.

To get started we needed two things,  a measurement of a successful conversion and 
a way to quantify which marketing methods were leading to the most conversions 
while maintaining our must-haves of everything being open-source and not-personally identifying.

### Defining Success

Since there are a variety of onboarding locations that draw new users to our site 
such as a tailored landing page from an ad, 
a blog post linked from a third party site like HackerNews or Twitter, 
or our home page from a search engine, 
defining a "success" is trickier than it first seems. 
There are some obvious win conditions such as:

> * Advertiser prospect fills out the "contact us" form on the Advertiser page 

Reviewer: ```suggestion
* An advertiser prospect fills out the "contact us" form on the Advertiser page 
```

  which leads to a conversion (new advertiser account and paid invoice)

> * Publisher prospect fills out the "contact us" form on the Publisher page 

Reviewer: ```suggestion
* A prospective publisher fills out the "contact us" form on the Publisher page 
```

  which leads to a conversion (new publisher account and approved to host paid ads)

but there are also harder to quantify successes such as:

> * Generic user educates themself on Surveilance Advertising in our Learning Hub

Reviewer: ```suggestion
* An interested visitor educates themself on Surveilance Advertising in our Learning Hub
```

> * Media-affiliated user fills out the generic "contact us" form

Reviewer: ```suggestion
* A media-affiliated user fills out the generic "contact us" form
```

> * Potential advertiser or publisher navigates to a page, browses for awhile, 

Reviewer: ```suggestion
* A potential advertiser or publisher navigates to a page, browses for awhile, 
```

For that we needed analytics.

### Open-source Analytics

After surveying a variety of open source analytics solutions, we decided to use Plausible. 
They have a great article on why one ought to choose [Plausible over Google Analytics](https://plausible.io/vs-google-analytics). 
With a simple onboarding process, they provide a light weight, open source option to monitoring inbound traffic source statistics 
in a privacy-preserving, aggregated format as well as tools for measuring successful conversions 
(which we defined as contact form submissions) per traffic source. 

All we had to do was add a distinct `?ref` tag for each ad url
to inform Plausible to aggregate the inbounds from that url as traffic from a distinct source.
We also added a `?ref` to the 'ads by EthicalAds' link underneath every placement
and try to include them on our content distributions as well.
This identifier is displayed to the user in the url and is in no way personally identifying.
Check the url for this article! Did you see it on twitter or hackernews? 
If so, the url should have a `?ref=twitter-ethical-funnel` or `?ref=hackernews-ethical-funnel` at the end

> which will go away once you explore the site as we don't track our users in our ads or locally. 

Reviewer: Technically, you don't need to do this since they're also recording the referrer. You do need this when multiple links from a site could send to you or when it sends from a site which doesn't send the referrer. Probably too much details...

Title: Building an Ethical Marketing Funnel
Date: August 31, 2021
description:
tags: marketing, privacy
authors: Ra Cohen
image: /images/posts/2021-marketing-funnel.jpg

> image_credit: <span>Photo by <a href="https://unsplash.com/@campaign_creators?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Campaign Creators</a> on <a href="https://unsplash.com/s/photos/funnel?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

Reviewer: Image looks great!

## The Problem

> Modern online marketing is invasive, annoying, and obtuse. The whole point of a marketing funnel is it's a continuous mission of getting, keeping and growing your customer base while using technology and other available tools to meet their needs more efficiently. No where in that definition does it say we must track individuals or hoard their personal information to weaponize it against them. We wanted our marketing funnel to be privacy-preserving, open-source, and transparent. In a word, Ethical.

Reviewer: Overall, I like this paragraph but I do feel the second sentence doesn't follow the first. The mission of acquiring, retaining, and growing your customer base doesn't follow that it is annoying or invasive (although it is sometimes). Something along the lines of "modern online marketing is an important part of growing any online business".

> ## What Is an Ethical Marketing Funnel?

Reviewer: ```suggestion
## What Is an Ethical Marketing Funnel and Why Would You Want One?
```

This is just a thought. I'm not sold on this heading myself but I thought it was interesting and could lead to a good discussion in the section.

> This post is assuming you are aware of what a marketnig funnel is and why you want one in the first place. If not, check out _____'s article on the topic of what is a marketing funnel and the various layers therein. To get started we needed two things,  a measurement of a successful conversion and a way to quantify which marketing methods were leading to the most conversions while maintaining our must-haves of everything being open-source and not-personally identifying.

Reviewer: ```suggestion
This post is assuming you are aware of what a marketing funnel is and why you want one in the first place. If not, check out _____'s article on the topic of what is a marketing funnel and the various layers therein. To get started we needed two things,  a measurement of a successful conversion and a way to quantify which marketing methods were leading to the most conversions while maintaining our must-haves of everything being open-source and not-personally identifying.
```

Reviewer: I do think this section should detail the *why* of a marketing funnel at least slightly. You want to know what's working in your business. How is your business going to grow? etc.

### Defining Success

A conversion for us is a simple foot in the door for our customer success team.

> Our conversions and thus the definition of a successful marketing strategy are when a user fills out one of our Contact Us forms. This establishes a relationship that did not previously exist and allows us to grow our user base. Our forms contain only relevant information to an individual's request, and is the only point where user-generated PII is collected, specifically their email address so that we may reply to their inquiry. This alone however does not inform us as to how a user arrived on our site or heard about us. Thus we decided to employ Analytics.

Reviewer: ```suggestion
Our conversions and thus the definition of a successful marketing strategy are when a user fills out one of our Contact Us forms. This establishes a relationship that did not previously exist and allows us to grow our user base. Our forms contain only relevant information to an individual's request, and is the only point where personal information is collected albeit voluntarily. Specifically, we request people's email address so that we may reply to their inquiry. This alone however does not inform us as to how a user arrived on our site or heard about us. Thus we decided to employ Analytics.
```

Reviewer: A graphic in this section would go a long way. However, we should talk about our dual funnel -- one for publishers and one for advertisers. For advertisers, our goal is to get people to talk to us about running a campaign. For publishers, our goal is to get them interested in hosting ethical ads although we still need to ensure they're a fit.

### Open-source Analytics

> After surveying a variety of open source analytics solutions, we decided to use Plausible. They have a great article on why one ought to choose [Plausible over Google Analytics](https://plausible.io/vs-google-analytics). With a simple onboarding process, they provide a light weight, open source option to tracking successful conversions as we defined as form submissions. But we were still missing a singular piece, a method for associating an inbound form with the marketing strategy that hooked them in.

Reviewer: I think we need to be explicit what we're using Plausible for. We have a bit of the why here but I think we need the what. Specifically, talk about our funnel of house ads and how we're measuring the conversion rate of house ads through to Plausible. I think that's cool stuff that people are interested in.

### One Transparent Cookie

> In order to properly give credit where credit is due, we thought long and hard about the most transparent, privacy-preserving method of satisfying our requirements of open source accountability with regards to accurately figuring out what marketing strategies are actually working. We decided to set up a singular, first-party, user-facing cookie displayed in the url to inform us of the referring link to our site. The code is available [here](https://github.com/readthedocs/ethicalads.io/blob/main/static-src/marketing.js) and is thus open source.

Reviewer: This doesn't clearly explain what this cookie is used for (sending the thing that referred somebody to our site). I think if we explained the house ads a bit more maybe this could make sense. Without already knowing how this works, I think folks reading this won't get it and they'll be confused.

Title: Building an Ethical Marketing Funnel
Date: September 30, 2021
description:
tags: marketing, privacy
authors: Ra Cohen
image: /images/posts/2021-marketing-funnel.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@campaign_creators?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Campaign Creators</a> on <a href="https://unsplash.com/s/photos/funnel?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

## The Problem

> When growing an online business, it's important to consider where your prospects are coming from and how to convert these prospects into customers. One key insight into this conversion comes from observing how your users interact with your site. Without some understanding of why people click on your page, what they do while they're there, and how this behavior corresponds to paying customers, you're at a serious disadvantage when it comes to optimizing your site to grow your business. What you need is a marketing funnel. A marketing funnel is a continuous mission of getting, keeping and growing your customer base while using technology to meet their needs more efficiently. However, many implementations of marketing funnels execute this mission by collecting and hoarding personally identifiable information of individual customers. This is by no means required by the definition nor is it even necessary as marketing funnels have long existed before web browser trackers let alone the internet. 

Reviewer: ```suggestion
When growing an online business, it's important to consider where your prospects are coming from and how to convert these prospects into customers. One key insight into this conversion comes from observing how your users interact with your site. Without some understanding of why people click on your page, what they do while they're there, and how this behavior corresponds to paying customers, you're at a serious disadvantage when it comes to optimizing your site to grow your business. Part of this process is building marketing funnel. A marketing funnel is a continuous mission of getting, keeping and growing your customer base while using technology to meet their needs more efficiently. However, many implementations of marketing funnels execute this mission by collecting and hoarding personally identifiable information of individual customers. This is by no means required by the definition nor is it even necessary as marketing funnels have existed long before web browser trackers let alone the internet. 
```

This section is *much* better!
By the way, if you space this out a little more line by line, it does make it a little easier to suggest a change to just a line instead of a whole paragraph. Not a big deal, but something worth considering.

> EthicalAds is a small start-up currently focusing on expanding our userbase which comes in two segments. We have potential advertisers who want to run new ad campaigns on our ad network and we have potential publishers who want to host ads on their websites/platforms in exchange for a lionshare of the generated ad revenue. In order to attract new users, we've created a number of advertiser/publisher focused house ads and advertiser/publisher specific landing pages in addition to our general referral sources of google or the "Ad served Ethically" text which runs beneath every ad. However, we previously had no good way of determining the relative success of these various approaches. In order to prioritize new ads or landing pages or other marketing strategies, we wanted data to guide our decision making to help us meet the needs of our customers as efficiently as possible. In other words, we needed a marketing funnel. Specifically, we wanted our marketing funnel to be privacy-preserving, open-source, and transparent. In a word, Ethical.

Reviewer: ```suggestion
EthicalAds is a small start-up currently focusing on expanding our userbase which comes in two segments. In the first segment, we have potential advertisers who want to run new ad campaigns on our ad network. In the second, we have potential publishers who want to host ads on their website or project in exchange for a lionshare of the generated ad revenue. In order to attract new users, we've created a number of advertiser/publisher focused house ads and advertiser/publisher specific landing pages in addition to our general referral sources of google or the "Ad served Ethically" text which runs beneath every ad. However, we previously had no good way of determining the relative success of these various approaches. In order to prioritize new ads or landing pages or other marketing strategies, we wanted data to guide our decision making to help us meet the needs of our customers as efficiently as possible. In other words, we needed a marketing funnel. Specifically, we wanted our marketing funnel to be privacy-preserving, open-source, and transparent. In a word, Ethical.
```

This section is also much better. It might jump the gun a little bit in talking about house ads (that's getting into the weeds a bit) but otherwise good. We want to discuss house ads and how they fit in the funnel, but the second paragraph might be early.

To get started we needed two things,  a measurement of a successful conversion and a way to quantify which marketing methods were leading to the most conversions while maintaining our must-haves of everything being open-source and not-personally identifying.

### Defining Success

> Since there are a variety of places that a new user can enter our site from such as a tailored landing page from a house ad, a blog post linked from a third party site like HackerNews or Twitter, or our home page from a search engine, defining a "success" is trickier than it first seems. There are some obvious win conditions such as 

Reviewer: ```suggestion
Since there are a variety of places that a new user can enter our site from such as a tailored landing page from a house ad, a blog post linked from a third party site like HackerNews or Twitter, or our home page from a search engine, defining a "success" is trickier than it first seems. There are some obvious win conditions such as:

```

Without the space, this doesn't render correctly.

* Advertiser prospect fills out the "contact us" form on the Advertiser page which leads to a conversion (new advertiser account and paid invoice)
* Publisher prospect fills out the "contact us" form on the Publisher page which leads to a conversion (new publisher account and approved to host paid ads)

> but there are also harder to quantify successes such as 

Reviewer: ```suggestion

but there are also harder to quantify successes such as:

```

* Generic user educates themself on Surveilance Advertising in our Learning Hub
* Media-affiliated user fills out the generic "contact us" form
* Potential advertiser or publisher navigates to a page, browses for awhile, and then fills out a "contact us" form

> Since this is our first marketing funnel, we decided to focus on only the first two scenarios as "successes" which is fairly simple since all of our contact us forms result in the same "form submitted" page when filled out. There are some drawbacks to this approach such as spammy form submissions being considered successes or even good leads for advertisers/publishers who simply didn't convert all of the way, but opening a line of dialogue between us and the user seems to be the largest hurdle right now. Hence defining a successful channel for communication as a success even if it doesn't necessarily always result in a conversion. Our forms contain only relevant information to an individual's request, and is the only point in our marketing funnel where user-generated PII is collected, specifically their name and email address so that we may reply to their inquiry. This alone however does not inform us as to how a user arrived on our site or how they heard about us. Thus we decided to employ Analytics.

Reviewer: If it is personally identifying (a name, email, IP), the user probably didn't "generate" it. Maybe just use "personally identifiable"?

### Open-source Analytics

> After surveying a variety of open source analytics solutions, we decided to use Plausible. They have a great article on why one ought to choose [Plausible over Google Analytics](https://plausible.io/vs-google-analytics). With a simple onboarding process, they provide a light weight, open source option to tracking aggregated inbound traffic source statistics as well as successful conversions (which we defined as contact form submissions) per traffic source. Being able to determine which ads or articles were generating not only the most traffic or form submissions, but also was generating the most successes proportionally to the amount of traffic was the vast majority of what we were looking for. Now we can A/B test new ads and landing pages for advertisers or publishers to determine which is better at convincing users to get in touch with us. We can also see which types of content are generating not only good amounts of traffic but also good resonance traffic, that is traffic which is comprised of potential advertisers or publishers. This is great! Mission Accomplished, right? Well, not quite. We still have the overwhelming issue of spam submissions. 

Reviewer: This section needs a bit more. I think we should talk about adding unique `?ref` codes to our house ads and the footer of our ads and measuring conversion rates of different landing pages.

Reviewer: This ended up sort of being in the next paragraph but it's sort of part of the same section (or maybe just the "one transparent cookie" header doesn't explain it).

> ### One Transparent Cookie

Reviewer: This title is still bad. It doesn't explain things.
